yew-functional/tests/lib.rs

```diff
@@ -377,9 +372,300 @@ fn use_effect_refires_on_dependency_change() {
     assert_eq!(result.as_str(), "11");
 }
 
+#[wasm_bindgen_test]
+fn use_context_scoping_works() {
```

Comment:
Having all the tests in the same file isn't great either, could you split them into separate files for each hook?

Reply:
Each hook is now in its own file and has its associated tests there as well.